### PR TITLE
Logentries and nodemailer integration

### PR DESCRIPTION
**Pull request by UnbounDev**
Now using [nodemailer](https://github.com/andris9/Nodemailer) to notify via email when either 1) hard error occurs 2) the api-sync server starts 3) the api-sync server stops. 

Additionally integrated w/ [logentries](https://logentries.com/) to channel significant events/errors/warnings to logentries service.

Refer to the README configuration section for details on setting up logentries and nodemailer tokens/auth.

### CODE REVIEW

File: config/config.template.js [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
-        //jquery: {minute: 5}
+  port: 8000,
+  output: 'data',
+  githubToken: '',
+  logentriesToken: '',
+  mailFrom: {
+    service: 'Gmail',
+    auth: {
+      user: '',
+      pass: ''
```
Comment (megawac) [inline config/config.template.js]: Can we not use a token?

Comment (jimaek) [inline config/config.template.js]: No way around the logentries token but for email lets use https://mandrill.com/ https://github.com/andris9/nodemailer-smtp-transport

**Comment by UnbounDev** [comment]
yea unfortunately need that token for the logentries service, switched to direct smtp usage: https://github.com/jsdelivr/api-sync/pull/17

File: config/config.template.js (hunk 1 above)
Comment (UnbounDev) [inline config/config.template.js]: yea unfortunately need that token for the logentries service, switched to direct smtp usage: #17

**Pull request merged**